mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -145,6 +164,15 @@ class LinearSVC @Since("2.2.0") (
   def setAggregationDepth(value: Int): this.type = set(aggregationDepth, value)
   setDefault(aggregationDepth -> 2)
 
+  /**
+   * Set optimizer for LinearSVC. Supported options: "lbfgs" and "owlqn".
+   *
+   * @group setParam
+   */
+  @Since("2.2.0")
+  def setOptimizer(value: String): this.type = set(optimizer, value.toLowerCase(Locale.ROOT))
```

Comment:
The solver is ```l-bfgs``` if training with lbfgs(l2 reg) or owlqn(l1/elasticNet reg) for other algorithms such as ```LinearRegression``` and ```LogisticRegression```, so I concerned that it may confuse users if we distinguish them in ```LinearSVC```. Or we should clarify it in document. Thanks.